go/bind/keybase.go

```diff
@@ -86,10 +86,11 @@ func Init(homeDir string, logFile string, runModeStr string, accessGroupOverride
 		Logs:         logs,
 	}
 
-	// FIXME (MBG): This is causing RPC responses to sometimes not be recieved
-	// on iOS. Repro by hooking up getExtendedStatus to a button in the iOS
-	// client and watching JS logs. Disabling until we have a root cause / fix.
 	kbfsParams := libkbfs.DefaultInitParams(kbCtx)
+
+	// Setting this flag will enable KBFS debug logging to alway be
+	// true in a mobile setting. Kill this setting if too spammy.
+	kbfsParams.Debug = true
```

Comment:
I made a mistake not putting these parameters settings in the goroutine, you think we could add them in this PR?